### DISCUSSION TITLE
Zelda the oricana of time

### DISCUSSION THREAD

**Discussion by DarkWolfWarr1or** [mupen64plus-ae/mupen64plus-ae#1074] [Q&A]
![Legend of Zelda, The - Ocarina of Time - Master Quest (E) (GC)  !  (1)](https://user-images.githubusercontent.com/117621941/200231626-fbfd7d70-e27e-45ee-b440-87f8f2bbb052.png)

Their should be a button to lower the platform but there isn't one, the button their is for the fire

**Comment by fzurita** [comment]
Hello, I suggest looking at a YouTube playthrough of the game. I don't believe there is an emulation bug here.
